desc/compute/_metric.py

```diff
@@ -1899,6 +1965,42 @@ def _gradzeta(params, transforms, profiles, data, **kwargs):
     return data
 
 
+@register_compute_fun(
+    name="g^aa",
+    label="g^{\\alpha \\alpha}",
+    units="m^{-2}",
+    units_long="None",
+    description="Fieldline bending term1",
+    dim=1,
+    params=[],
+    transforms={},
+    profiles=[],
+    coordinates="rtz",
+    data=["grad(alpha)"],
+)
+def _g_sup_aa(params, transforms, profiles, data, **kwargs):
+    data["g^aa"] = dot(data["grad(alpha)"], data["grad(alpha)"])
+    return data
+
+
+@register_compute_fun(
+    name="g^ra",
+    label="g^{\\rho \\alpha}",
+    units="~",
+    units_long="None",
+    description="Fieldline bending term2",
+    dim=1,
+    params=[],
+    transforms={},
+    profiles=[],
+    coordinates="rtz",
+    data=["grad(alpha)", "e^rho"],
+)
+def _g_sup_ra(params, transforms, profiles, data, **kwargs):
+    data["g^ra"] = dot(data["grad(alpha)"], data["e^rho"])
```

Comment:
This will be well-defined at magnetic axis with $\psi, \alpha$ Clebsch coordinates, whereas they'll blow up with $\rho, \alpha$. In case its useful to have finite stability quantities near axis for optimization. Mentioning again because don't remember if you decided for /against it

Reply:
I don't care about near axis behavior